main.go

```diff
@@ -48,17 +53,27 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	utilruntime.Must(crunchybridgev1alpha1.AddToScheme(scheme))
-	utilruntime.Must(dbaasredhatcomv1alpha1.AddToScheme(scheme))
-	utilruntime.Must(dbaasoperator.AddToScheme(scheme))
 	//+kubebuilder:scaffold:scheme
 }
 
+type mainConfig struct {
+	apiURL    string
+	clientset *kubernetes.Clientset
+}
+
+var dbaasInit func(ctrl.Manager, mainConfig)
+
 func main() {
-	var metricsAddr string
+	// Variables from boilerplate
+	var metricsAddr, probeAddr string
 	var enableLeaderElection bool
-	var probeAddr string
+
 	var crunchybridgeAPIURL string
-	flag.StringVar(&crunchybridgeAPIURL, "crunchybridgeapi-url", "https://api.crunchybridge.com", "the Crunchy bridge API URL (no slash in the end).")
+	// Namespace and Name for APIKey secret default values
+	credNamespace := "default"
```

Comment:
why adding default namespace for credentials? we can pass in CR reference as done in inventory CR?

Reply:
The credentials are wherever they are created, this is a variable that holds the environment-configured location (namespace and name) of where the credentials live. Using a CR to bootstrap the DBaaS system is a design decision for the DBaaS approach to providers but is not yet necessary for a non-DBaaS operator.

Reply:
understood thanks, can be added these environment variables for a container like https://github.com/CrunchyData/crunchy-bridge-operator/blob/master/config/manager/manager.yaml#L56 WDYT?